### PR TITLE
DOCS-7780 - OpenTelemetry for AWS Lambda

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do? What is the motivation?
Adds an OpenTelemetry page to the Serverless > AWS Lambda section.

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

Open questions:
- Are there any other options that should be documented?
- What are the high-level reasons one option would be used over another?
- Please review the code samples in the OTel SDK option, as they've been modified significantly

### CODE REVIEW

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+[OpenTelemetry][1] is an open source observability framework that provides IT teams with standardized protocols and tools for collecting and routing telemetry data.
+
+This page discusses using OpenTelemetry with Datadog Serverless Monitoring for AWS Lambda. For more information, including how to use OpenTelemetry in non-serverless environments, see [OpenTelemetry in Datadog][2].
+
+## Instrument AWS Lambda with OpenTelemetry
+
+There are multiple ways to instrument AWS Lambda functions with OpenTelemetry and send the data to Datadog:
+
+- [OpenTelemetry API support within the Datadog tracers](#opentelemetry-api-support-within-datadog-tracers) (recommended)
+- [OpenTelemetry SDK within the Datadog Agent](#opentelemetry-sdk-within-the-datadog-agent) (beta)
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/opentelemetry.md:19]:
```suggestion
- [Send OpenTelemetry Traces from any OpenTelemetry SDK through the Datadog Lambda Extension](#opentelemetry-sdk-within-the-datadog-agent) (beta)
```

Comment (sumedham) [reply content/en/serverless/aws_lambda/opentelemetry.md:19]: Open to other syntax here, I just felt like this was more straightforward

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+### OpenTelemetry API support within Datadog tracers
+
+The Datadog tracing library, which is included in the Datadog Lambda Extension upon installation, accepts the spans and traces generated by OpenTelemetry-instrumented code, processes the telemetry, and sends it to Datadog.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/opentelemetry.md:23]:
```suggestion
The Datadog tracing library, which is included in the Datadog Lambda Extension upon installation, accepts custom spans and traces created with OpenTelemetry-instrumented code, processes the telemetry, and sends it to Datadog.
```

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+You can use this approach if, for example, your code has already been instrumented with the OpenTelemetry API. You may also use this approach if you want to instrument using vendor-agnostic code with the OpenTelemetry API while still gaining the benefits of using the Datadog tracing libraries.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/opentelemetry.md:25]:
```suggestion
You can use this approach if, for example, your main goal is to code has already been instrumented with the OpenTelemetry API. This means you can maintain vendor-neutral instrumentation of all your services, while still taking advantage of Datadog’s native implementation, tagging, and, features. 
```

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+To instrument AWS Lambda with the OpenTelemetry API, set the environment variable `DD_TRACE_OTEL_ENABLED` to `true`.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/opentelemetry.md:27]:
```suggestion
To instrument AWS Lambda with the OpenTelemetry API, set the environment variable `DD_TRACE_OTEL_ENABLED` to `true` in your Lambda function and see [Custom instrumentation with the OpenTelemetry API][3]  for runtime-specific instructions.
```

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+See [Custom instrumentation with the OpenTelemetry API][3] for more details.
+
+### OpenTelemetry SDK within the Datadog Agent
+
+<div class="alert alert-warning">This feature is in beta.</div>
+
+This approach is analogous to [OLTP Ingest in the Datadog Agent][4].
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/opentelemetry.md:35]:
```suggestion
This approach is analogous to [OLTP Ingest in the Datadog Agent][4]. It is recommended in situations where tracing support may not be available for your runtime (e.g. Rust or PHP). 
```

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+1. Tell OpenTelemetry to export spans to the Datadog Lambda Extension. Then, add OpenTelemetry's instrumentation for AWS Lambda. This is analogous to adding the tracing layer.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/opentelemetry.md:37]: Customers may not know what "this is analagous to the tracing layer" means. Should we just remove it?

**Review by sumedham — CHANGES_REQUESTED** [review]
Some minor positioning comments

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+### OpenTelemetry API support within Datadog tracers
+
+The Datadog tracing library, which is included in the Datadog Lambda Extension upon installation, accepts the spans and traces generated by OpenTelemetry-instrumented code, processes the telemetry, and sends it to Datadog.
+
+You can use this approach if, for example, your code has already been instrumented with the OpenTelemetry API. You may also use this approach if you want to instrument using vendor-agnostic code with the OpenTelemetry API while still gaining the benefits of using the Datadog tracing libraries.
+
+To instrument AWS Lambda with the OpenTelemetry API, set the environment variable `DD_TRACE_OTEL_ENABLED` to `true`.
+
+See [Custom instrumentation with the OpenTelemetry API][3] for more details.
```
Comment (cswatt) [inline content/en/serverless/aws_lambda/opentelemetry.md:29]:
```suggestion
```

File: content/en/serverless/aws_lambda/opentelemetry.md (hunk 6 above)
Comment (cswatt) [inline content/en/serverless/aws_lambda/opentelemetry.md:37]:
```suggestion
1. Tell OpenTelemetry to export spans to the Datadog Lambda Extension. Then, add OpenTelemetry's instrumentation for AWS Lambda.
```

File: content/en/serverless/aws_lambda/opentelemetry.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+
+### OpenTelemetry SDK within the Datadog Agent
```
Comment (cswatt) [inline content/en/serverless/aws_lambda/opentelemetry.md:31]:
```suggestion
### Send OpenTelemetry traces from any OpenTelemetry SDK through the Datadog Lambda Extension {#sdk}
```

**Pull request merged**